pkg/types/aws/defaults/platform.go

```diff
@@ -4,6 +4,24 @@ import (
 	"github.com/openshift/installer/pkg/types/aws"
 )
 
+var (
+	defaultMachineClass = map[string]string{
+		"eu-north-1":    "m5",
+		"eu-west-3":     "m5",
```

Comment:
I don't believe RHEL has signed off on the M5 series. We will need to figure out which machine class we want to use as a fallback.

Reply:
> I don't believe RHEL has signed off on the M5 series. We will need to figure out which machine class we want to use as a fallback.

I thought someone tested m5 somewhere :p.  But whatever, it's easy to slot in a different alternative if you know what it should be.

Reply:
Eric and Clayton think we should go with M5 for now. Ignore me!